controls/sidedrawer/transitions.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Transitions
+page_title: .NET MAUI SideDrawer Documentation | Transitions
+description: Check our &quot;Transitions&quot; documentation article for Telerik UI for .NET MAUI SideDrawer control.
+position: 3
+slug: sidedrawer-features-transitions
+---
+
+# Transitions
+
+Transitions are the animation effects applied to the side drawer while it is being opened and closed.
+
+* `DrawerTransitionDuration` (`double`)&mdash; Defines the duration of the chosen transition.
```

Comment:
remove the space between the property name and its type and the dash and explanation